dashboard/test/models/pd/workshop_material_order_test.rb

```diff
@@ -301,4 +301,18 @@ class Pd::WorkshopMaterialOrderTest < ActiveSupport::TestCase
     order.refresh
     assert_nil order.tracking_id
   end
+
+  test 'orders are still valid if the user is deleted' do
```

Comment:
No changes expected, but grumble, grumble. The `assert_equal` is not part of the test asserts, it is asserting a precondition. And this is two tests, one confirming that the order is valid with a deleted user, another confirming that the order is invalid after a user change.